server.js

```diff
@@ -1,20 +1,128 @@
 const express = require("express");
 const cors = require("cors");
-
+var validator = require("email-validator");
+var moment = require("moment");
 const app = express();
 
+const port = process.env.PORT || 3005;
 app.use(express.json());
 app.use(cors());
 
 //Use this array as your (in-memory) data store.
 const bookings = require("./bookings.json");
+const { request } = require("express");
+
+// read all bookings
+app.get("/bookings", function (request, response) {
+  response.json(bookings);
+});
+
+// create a new booking
+
+app.post("/bookings", (request, response) => {
+  let indexOfLastBooking = bookings.length - 1;
+  console.log(indexOfLastBooking);
+  let bookingsId = [];
+  for (let i = 0; i < bookings.length; i++) {
+    bookingsId.push(bookings[i].roomId);
+  }
+  console.log(bookingsId);
+  let newBooking = {
+    id: bookings[indexOfLastBooking].id + 1,
+    title: request.body.title,
+    firstName: request.body.firstName,
+    surname: request.body.surname,
+    email: request.body.email,
+    roomId: request.body.roomId,
+    checkInDate: request.body.checkInDate,
+    checkOutDate: request.body.checkOutDate,
+  };
+  if (!newBooking.title || newBooking.title.length === 1) {
```

Comment:
While this might work. Too many chained if conditions is not good standards. Clean code encourage having a better solution as many of the return are similar 
so you can either make switch case 
or make a loop for in